jdk/src/solaris/bin/zero/jvm.cfg

```diff
@@ -31,5 +34,8 @@
 # "-XXaltjvm=<jvm_dir>" option, but that too is unsupported
 # and may not be available in a future release.
 #
--server KNOWN
--client IGNORE
+-j9vm KNOWN
+-hotspot IGNORE
```

Comment:
Although most of the jvm.cfg files contain them, these lines for `-hotspot, -classic, -native, -green` seem to have been copied from some really old build. When we updated the jvm.cfg for Windows, I suggested minimal changes, i.e. set `-server IGNORE` and add `-j9vm KNOWN`.

Reply:
I mostly left them in because they were there on the `s390x/jvm.cfg` that was already in the repository. I'll remove them